modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/tests/ProcessBuilderKaleoDesignerReact.testcase

```diff
@@ -4611,7 +4611,7 @@ definition {
 
 		Button.clickPublish();
 
-		Alert.viewErrorMessage(errorMessage = "The Start node must have an outgoing transition");
+		Alert.viewErrorMessage(errorMessage = "The Task node must have an incoming transition");
```

Comment:
By changing the error message; the test isn't really confirming _StartNodeMustHaveOutgoingTransition_ anymore. Is there a way to change the test steps to assert the original error?

Reply:
You are sure. I hadn't noticed that.